lib/takeaway.rb

```diff
@@ -0,0 +1,28 @@
+class Takeaway
+
+  attr_reader :menu, :order
+
+  def initialize
+    @menu = { :dish1 => 1,
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)